transformers/insolvency_resource.go

```diff
@@ -88,9 +88,11 @@ func PractitionerResourceDaosToPractitionerFilingsResponse(practitionerResourceD
 			appointedPractitionerResource = models.AppointedPractitionerResource{
```

Comment:
Not sure if the practitionersResourceDao should have an appointment. Is it not the links that should containe the appointment?